### ISSUE TITLE
SCAM link to download

### ISSUE THREAD

**Issue by dexter2040** [ChrisAnd1998/TaskbarX#1064]
I haven't seen any other related issue, and I find this is the best way to indicate that problem.

Someone got registered domain: https://taskbarx.org/ and offers "Free" download. No info that is created by @ChrisAnd1998, just a markdown to some company in USA, and only in links, we can find a link to a Chris's GitHub.

**Comment by mohammedhidji** [comment]
I downloaded the app from there thinking it was the official website, is it safe though? I ran it through multiple antivirus software and nothing showed up as malicious.

**Comment by dexter2040** [comment]
As long the author confirms it belongs to him, or he's aware about it, I would not recommend using it.
The whole source of the app is available publicly, so anyone could implement any additional code.

**Comment by dexter2040** [comment]
As long the author confirms it belongs to him, or he's aware about it, I would not recommend using it.
Whole source of the app is available publicly, so anyone could implement any additional source.

**Comment by skifli** [comment]
> I haven't seen any other related issue, and I find this is the best way to indicate that problem.
> 
> Someone got registered domain: https://taskbarx.org/ and offers "Free" download. No info that is created by @ChrisAnd1998, just a markdown to some company in USA, and only in links, we can find a link to a Chris's GitHub.

Yeah I already reported this a few months back in #1054, no response yet though.
